prow/test/integration/config/prow/cluster/webhook_server_service.yaml

```diff
@@ -18,7 +18,27 @@ metadata:
   labels:
     app: webhook-server
   namespace: default
-  name: prowjob-validation-webhook
+  name: prowjob-validating-webhook
```

Comment:
not sure but my feeling is that this could be done by a single service, am I miss something?